tools/stimulus/functions-R.yaml

```diff
@@ -26,9 +26,6 @@ igraph_delete_vertices:
 igraph_ecount:
     IGNORE: RR, RC
 
-igraph_neighbors:
-    IGNORE: RR, RC
-
```

Comment:
I thought @krlmlr wanted all of these to appear explicitly in `functions-R.yaml`, even though they're in the upstream `functions.yaml`.

@krlmlr What's your preference?

Reply:
It seems @krlmlr prefers not to list them: https://github.com/igraph/rigraph/pull/2099#discussion_r2339498557